demo/site/src/common/blocks/TextLinkBlock.tsx

```diff
@@ -0,0 +1,12 @@
+"use client";
+import { PropsWithData, withPreview } from "@comet/cms-site";
+import { TextLinkBlockData } from "@src/blocks.generated";
+
+import { LinkBlock } from "./LinkBlock";
+
+export const TextLinkBlock = withPreview(
+    ({ data: { link, text } }: PropsWithData<TextLinkBlockData>) => {
+        return <LinkBlock data={link}>{text}</LinkBlock>;
```

Comment:
The (now deleted) old text link block had a special handling for DAM file downloads to show the file size. Please restore this behavior.